styles.css

```diff
@@ -98,3 +98,99 @@ h1 {
   top: 100px;
   left: 120px;
 }
+/*
+.left_arm:hover{
+  transform:  translate(-40px, -60px) rotate(120deg) ;
+  transition: all 1s;
+}
+.right_arm:hover{
+  transform:translate(40px, -60px) rotate(-120deg);
+  transition: all 1s;
+} 
+*/
+.left_arm{
+  animation: wave_left 1.5s infinite;
+}
+@keyframes wave_left {
+  0% {
+  transform:rotate(0deg) ;
+  }
+  100% {
+  transform:  translate(0px, -40px) rotate(120deg) scale(0.9) ;
+   background-color:  rgb(84, 173, 84);;  
+}
+}
+.right_arm{
+  animation: wave_right 1.5s infinite;
+}@keyframes wave_right {
+  0% {
+  transform:rotate(0deg) ;
+  }
+  100% {
+  transform:  translate(0px, -40px) rotate(-120deg) scale(0.9) ;
+  background-color: rgb(84, 173, 84);;
+}
+}
+.right_eye{
+  animation: eye 1.5s infinite;
+}
+.left_eye{
+animation: eye 1.5s  infinite;
+}
+@keyframes eye {
+  0% {
+  transform: translate(0px);
+  }
+  100% {
+height: 0px;
+transform: translate(0px,7.5px) scale(1.6); 
+  }
+}
+.head{
+  animation:head 1.5s infinite;
+}
+@keyframes head{
+  0% {
+transform: translate(0px);
```

Comment:
I don't think this line is necessary since it will not make any changes to the movement of the elements.

Reply:
I know, but can it be left blank?

Reply:
yes, you can delete it, and keep the 100%.